grakn-test-tools/src/main/java/ai/grakn/factory/GraknSessionTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2016  Grakn Labs Limited
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+
+package ai.grakn.factory;
+
+import ai.grakn.Keyspace;
+import ai.grakn.util.CommonUtil;
+import ai.grakn.util.ErrorMessage;
+import ai.grakn.util.GraknTestUtil;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * <p>
+ *     Test {@link ai.grakn.GraknSession}
+ * </p>
+ *
+ * <p>
+ *     A {@link ai.grakn.GraknSession} used for testing which bypasses the need to hit the REST controller responsible
+ *     for feeding in the appropriate config file.
+ *
+ *     This session can be used to provide {@link ai.grakn.GraknTx}'s dependent on either a TinkerGraph or
+ *     JanusGraph
+ * </p>
+ *
+ * @author Filipe Peliz Pinto Teixeira
+ */
+public class GraknSessionTest extends GraknSessionImpl{
```

Comment:
I think this needs a different name. I assumed this would contain tests, but it's actually a utility class.

Reply:
Hmmmm good point. . . Will have to think of one.